code/src/js components/Button1.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import "css components/button1.css";
+
+export const Button1 = () => {
```

Comment:
I think it's a bit vulnerably to name them Button1 etc. Maybe not in this case when the site is shallow but in a major project where it's a lot of buttons it can be hard to remember which button belongs to which name of it. Then it might be better to call them submitButton etc, like you do in the classname :) But this wouldn't be a problem if you gathered them in one file like I suggested before.